src/docs/sdk/research/performance/index.mdx

```diff
@@ -192,6 +192,38 @@ In the example above, if an error bubbles up the call stack we want to be able t
 
 All those different expectations makes it hard to reuse, in an understandable way, the current notion of `scope`, how breadcrumbs are recorded, and how those different concepts interact.
 
+Finally, it is worth noting that the changes to restructure scope management most likely cannot be done without breaking existing SDK APIs. Existing SDK concepts, like hubs, scopes, breadcrumbs, user, tags, contexts, would all have to be remodeled.
+
 ## Span Ingestion Model
 
-Coming soon.
+Consider a trace depicted by the following span tree:
+
+```
+F
```

Comment:
Good that you mention... here I intended to play with the ambiguity, but didn't yet fully...
The "button click" as I described later is not typically a transaction.

In OTel's model, it would/could be just a regular span. In Sentry's model, if the span is not a transaction or there is no ancestor span that is a transaction, then it won't be ingested (but the other `B` and `S` spans would still be ingested!).

Reply:
My original intention was just a hypothetical "trace", not necessarily one we could reproduce with Sentry.
But considering that 

> Spans that are at the entry point to a given service are marked with a `*` to denote that they are transactions.

The above is already a Sentry-specific requirement, then, yes, let's make this `F*`.

Pending a short item later about Frontend Transactions...